_meta/beat.reference.yml

```diff
@@ -70,6 +70,14 @@ apm-server:
       # a matching index pattern needs to be specified here.
       #index_pattern: "apm-*"
 
+  # golang expvar support - https://golang.org/pkg/expvar/
+  #expvar:
+  # Set to true to Expose expvar
+  #enabled: false
```

Comment:
Should `enabled` and `url` be in the `expvar` namespace` if yes, it needs 2 additional spaces.

Reply:
You are right, but it will be fixed with https://github.com/elastic/apm-server/pull/640/files#diff-386819a19a12310e5e0ea62392099482R73